sw/host/opentitanlib/src/app/config/servo_micro.json

```diff
@@ -0,0 +1,27 @@
+{
+  "interface": "servo_micro",
+  "pins": [
+    {
+      "name": "RESET",
+      "level": true,
+      "alias_of": "IO_EXP_16"
+    }
+  ],
+  "strappings": [
+    {
+      "name": "RESET"
```

Comment:
I think the RESET pin should be mentioned in this RESET strapping.  It is probably the case that the "rescue" bootstrap protocol uses the RESET pin by name, but probably other features of opentitantool uses the strapping called "RESET", and the rescue protocol implementation could be aligned in the future.

In the majority of cases, the RESET strapping consists simply of setting the level of the RESET pin to "false" (low).  Sometimes, other bootstrap pins are set to particular values, possibly to make sure the chip goes into the desired default mode.

Reply:
I added the pin to `RESET` strap to follow existing pattern and I upload #19171  for C2D2